From b044352ea4607bf33e6699b34a71f6f83708f694 Mon Sep 17 00:00:00 2001
From: Matt Mitchell <mmitche@microsoft.com>
Date: Mon, 9 Sep 2019 15:43:45 -0700
Subject: [PATCH 09/11] Switch IsShipping -> IsShippingPackage (#3909)

It was brought up that IsShippingPackage is more correct here (IsShipping sets IsShippingPackage for packages).
Also invert the conditional to be more in line with the convention.
---
 .../tools/Version.BeforeCommonTargets.targets                   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets b/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets
index 07f8004a..0808c222 100644
--- a/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets
+++ b/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets
@@ -101,7 +101,7 @@
     <!--
       If a project produces non-shipping packages, these packages should always include the build number label
     -->
-    <VersionSuffix Condition="'$(IsShipping)' == 'false'">$(_PreReleaseLabel)$(_BuildNumberLabels)</VersionSuffix>
+    <VersionSuffix Condition="'$(IsShippingPackage)' != 'true'">$(_PreReleaseLabel)$(_BuildNumberLabels)</VersionSuffix>
 
     <!--
       Disable NuGet Pack warning that the version is SemVer 2.0.
-- 
2.18.0

